zipline/algorithm.py

```diff
@@ -410,9 +410,6 @@ def noop(*args, **kwargs):
         # target/delta of the capital changes, along with values
         self.capital_changes = kwargs.pop('capital_changes', {})
 
-        # A dictionary of the actual capital change deltas, keyed by timestamp
-        self.capital_change_deltas = {}
```

Comment:
Correct, I had previously saved the deltas as I thought we might use them, but this attribute doesn't actually get referenced anywhere